astropy/table/serialize.py

```diff
@@ -71,6 +72,9 @@
     "astropy.coordinates.representation.RadialDifferential",
     "astropy.coordinates.representation.PhysicsSphericalDifferential",
     "astropy.coordinates.representation.CylindricalDifferential",
+    "astropy.coordinates.angles.Latitude",
+    "astropy.coordinates.angles.Longitude",
+    "astropy.coordinates.angles.Angle",
```

Comment:
Do these have to be listed twice?

Reply:
This is for backwards compatibility with files written with the old definitions. We really should rewrite this to just allow any class that is defined in astropy, or have a regex that has a `*` or so. Anyway, for this PR this sadly cannot be helped (same for the representations...)